_shared/project/Makefile

```diff
@@ -5,11 +5,11 @@ help = help::; @echo $$$$(tput bold)$(strip $(1)):$$$$(tput sgr0) $(strip $(2))
 $(call help,make help,print this help message)
 
 .PHONY: services
-{% if cookiecutter.get("services") == "yes" %}
+{% if cookiecutter.get("postgres") == "yes" or include_exists("docker-compose/services.yml") %}
```

Comment:
Every project always gets a do-nothing `make services` command that is missing from `make help`. If _either_ `"postgres"` is `"yes"` in `cookiecutter.json` _or_ `.cookiecutter/includes/docker-compose/services.yml` then `make services` gets filled out with an actual `docker compose` command and a `make help` string.